app/templates/buyers/_requirements_meta.html

```diff
@@ -0,0 +1,13 @@
+<div id="requirements-meta">
+  <h2 class="sidebar-heading">Published</h2>
+  <p class="sidebar-content">{{ brief.publishedAt|dateformat }}</p>
+  {% if brief.status == 'closed' %}
+    <h2 class="sidebar-heading">Closed</h2>
+    <p class="sidebar-content">{{ brief.applicationsClosedAt|dateformat }}</p>
+  {% else %}
+    <h2 class="sidebar-heading">Closing</h2>
+    <p class="sidebar-content">{{ brief.applicationsClosedAt|dateformat }}</p>
+  {% endif %}
+  <h2 class="sidebar-heading">Framework</h2>
+  <p class="framework-name"><a href="{{ framework_agreement_url }}">{{ brief.frameworkName }}</a></p>
```

Comment:
I was mostly cribbing off the meta stuff on the g-cloud page, and it's done like this there.

I think it would actually make my tests easier if the classes were the same.

Change or keep...your call?